docs/src/Guides/01 Getting Started.md

```diff
@@ -9,8 +9,30 @@ Hi! So you want to make a bot powered by snakes. This guide aims to get you star
 - [x] [A bot account](02 Creating Your Bot.md)
 - [ ] An aversion to snakes
 
-## Installation
+## Using a Template
```

Comment:
imo the two setup methods should be in content tabs for clarity, rn the page looks a bit cluttered